code/modules/clothing/clothing.dm

```diff
@@ -190,7 +191,7 @@ BLIND     // can't see anything
 	max_heat_protection_temperature = SPACE_SUIT_MAX_TEMP_PROTECT
 	strip_delay = 80
 	put_on_delay = 80
-
+	var/alt_desc = null

```

Comment:
put this on /obj/item/clothing, no point duplicating it on space suits and space helmets.